docs/resources/packer_channel.md

```diff
@@ -11,45 +11,42 @@ The Packer Channel resource allows you to manage image bucket channels within an
 
 ## Example Usage
 
-To create a channel with no assigned iteration.
+To create a channel.
 ```terraform
 resource "hcp_packer_channel" "staging" {
   name        = "staging"
   bucket_name = "alpine"
 }
 ```
 
-To create, or update an existing, channel with an assigned iteration.
+To create a channel with a Terraform-managed iteration.
 ```terraform
 resource "hcp_packer_channel" "staging" {
   name        = "staging"
   bucket_name = "alpine"
   iteration {
-    id = "iteration-id"
+    # Exactly one of `id`, `fingerprint` or `incremental_version` must be passed
+    id = "01H1SF9NWAK8AP25PAWDBGZ1YD"
+    # fingerprint = "01H1ZMW0Q2W6FT4FK27FQJCFG7"
+    # incremental_version = 1
   }
 }
 
-# Update assigned iteration using an iteration fingerprint
+# To configure a channel to have no assigned iteration, use a "zero value".
+# The zero value for `id` and `fingerprint` is `""`; for `incremental_version`, it is `0`
 resource "hcp_packer_channel" "staging" {
   name        = "staging"
   bucket_name = "alpine"
   iteration {
-    fingerprint = "fingerprint-associated-to-iteration"
-  }
-}
-
-# Update assigned iteration using an iteration incremental version
-resource "hcp_packer_channel" "staging" {
-  name        = "staging"
-  bucket_name = "alpine"
-  iteration {
-    // incremental_version is the version number assigned to a completed iteration.
-    incremental_version = 1
+    # Exactly one of `id`, `fingerprint` or `incremental_version` must be passed
+    id = ""
+    # fingerprint = ""
+    # incremental_version = 0
```

Comment:
Would an empty block have the same result?  I guess the fields inside become at least one required, right? 
```
iteration {}
```

Reply:
At least one field is required